CHANGELOG.md

```diff
@@ -6,6 +6,35 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### 4.2.0 - 2019-05-31
+
+### Added
+
+- Check execute PMA on instruction frontend
+- Add support for non-contiguous cacheable regions to the PMA checks
+- Provision exponential backoff for AMO SC in L1 D$ miss handler
+
+### Changed
+
+- Several small fixes to get the code running on VCS
+- Fix compressed instruction decoding in tracer
+- Fix privilege bug in performance counters. The counters have always been accessible in user mode.
+- Fix RISC-V PK simulation bug caused due to insufficient time to init the `a0` and `a1` registers via the bootrom
+- Fix bug in `wt_axi_adapter` (only appeared when dcache lines were wider than icache lines)
+- Fix potentially long timing path in `axi_lite_interface`
+- Fix VCS elab warning in `load_store_unit`
+- Replace PLIC with implementation from lowRISC
+- Re-work interrupt and debug subsystem to associate requests during decode. This improves stability on for non-idempotent loads.
+- Bump `common_cells` to `v1.13.0`
+- Bump `riscv-dbg` to `v0.1` and `fpnew` to `v0.5.1`
```

Comment:
Please bump `fpnew` to `v0.5.3` which fixes a bug introduced with the FPGA pipeline options.

Reply:
Thanks for letting us now. I bumped the submodule.